src/sentry/static/sentry/app/views/discover/discover.tsx

```diff
@@ -471,12 +456,7 @@ export default class Discover extends React.Component<Props, State> {
 
         <DiscoverGlobalSelectionHeader
           organization={organization}
-          projects={currentQuery.projects}
```

Comment:
This gets overwritten by https://github.com/getsentry/sentry/blob/5743740b65d4d26a656cbb0f3481494ceead826d/src/sentry/static/sentry/app/utils/withProjectsSpecified.tsx#L49